File: code/src/test/scala/cern/colt/matrix/tint/impl/SparseCCMIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseCCMIntMatrix2DViewTest(arg0: String) extends SparseCCMIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseCCMIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseCCMIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseCCMIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/SparseCCMIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class SparseCCMIntMatrix2DViewTest extends SparseCCMIntMatrix2DTest {

    public SparseCCMIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCMIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCMIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCMIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
